gems/pending/Gemfile

```diff
@@ -24,7 +24,7 @@ gem "fog",                     "~>2.0.0.pre.0",     :require => false
 gem "fog-core",                "!=1.31.1",          :require => false
 gem "httpclient",              "~>2.5.3",           :require => false
 gem "kubeclient",              "=0.8.0",            :require => false
-gem "hawkular-client",                    :require => false, :git => "git@github.com:hawkular/hawkular-client-ruby.git", :branch => "hawkular-all"
+gem "hawkular-client",                    :require => false, :git => "git@github.com:hawkular/hawkular-client-ruby.git", :branch => "master"
```

Comment:
@simon3z I added a ton of stuff and re-shuffled in the hawkular ruby client. While spec tests were not modified and they all pass, I still thought it would be good if you could test against your code to see if all it ok.